# Data Science Tools

### James Z

##### SUNY NP

##### I am interested in Data Science because I want to reveal the hidden trends in any popular events and become important and able to assist others.

# The below should print "hello world"

In [4]:
print("hello world")

hello world


##### In this notebook, Data Science Tools will be introduced. This is the final assigment of the week 2 course "Tools for Data Science" 1/13/2024

#### Some of the most used tools for data scientists are:
####
##### 1. RStudio
##### 2. Python
##### 3. Jupyter Notebook
##### 4. Java
##### 5. Github

#### Some of the commonly used libraries used by Data Scientists include: If we use python the most common are:
####
##### 1. Matplotlib
##### 2. Numpy
##### 3. Tensor Flow
##### 4. Scikit-learn

#### If we use R, the most common tools are:
####
##### 1. ggplot
##### 2. dplyr
##### 3. stringr

In [15]:
print("I love Data Science")

I love Data Science


### Below will show a few examples of evaluating arithmetric expression in Python:

###### This a simple arithmetic expression to mutiply then add integers that is implemented in Python

In [14]:
((4+6)*4)*2

80

##### In the following exercise is presented an arithmetic expression to convert 200 minutes to hours by diving by 60.

In [2]:
200/60

3.3333333333333335

##### 200 minutes can be converted to 3.333...hours

#### Objectives in this notebook:
######
##### 1. practice markdown in Jupyter Notebook
##### 2. be similar with the most common used tools in Data Science
##### 3. implement simple arithmetric expression in Python
##### 4. knowing how to make different types of markdowns in Jupyter Notebooks

In [1]:
print("Shilong Zhou")

Shilong Zhou


## Author - James Z